### PR TITLE
Brightwheel tweaks

### PR DESCRIPTION
**Pull request by GuillaumeDelente**
- Always have selected view centered
- Remove joda time dependency

### CODE REVIEW

File: horizontal-picker/src/main/java/com/github/jhonnyx2012/horizontalpicker/HorizontalPickerRecyclerView.java [hunk 1]
```diff
@@ -1,149 +1,159 @@
...
+    post(new Runnable() {
+      @Override
+      public void run() {
+        layoutManager.startSmoothScroll(smoothScroller);
+      }
+    });
+  }
+
+  public void setDate(LocalDate date) {
+    Day firstDay = adapter.getItem(OFFSET);
```
Comment (john-oleary) [inline horizontal-picker/src/main/java/com/github/jhonnyx2012/horizontalpicker/HorizontalPickerRecyclerView.java]: could you add more detail about which each `offset` is?

File: horizontal-picker/src/main/java/com/github/jhonnyx2012/horizontalpicker/HorizontalPickerRecyclerView.java [hunk 2]
```diff
@@ -1,149 +1,159 @@
...
+
+  public HorizontalPickerRecyclerView(Context context, @Nullable AttributeSet attrs) {
+    super(context, attrs);
+  }
+
+  public HorizontalPickerRecyclerView(Context context, @Nullable AttributeSet attrs, int defStyle) {
+    super(context, attrs, defStyle);
+  }
+
+  public void init(Context context,
```
Comment (john-oleary) [inline horizontal-picker/src/main/java/com/github/jhonnyx2012/horizontalpicker/HorizontalPickerRecyclerView.java]: Is there a way to add more inertia on the scroll view?   If I swipe, it scrolls very far in the list, doesn't seem to slow down

File: horizontal-picker/src/main/java/com/github/jhonnyx2012/horizontalpicker/HorizontalPickerRecyclerView.java (hunk 1 above)
Comment (GuillaumeDelente) [reply horizontal-picker/src/main/java/com/github/jhonnyx2012/horizontalpicker/HorizontalPickerRecyclerView.java]: Yep, this is the number of "dummy" views that we need to place in the adapter at the beginning of the list and at the end of the list in order to have the following behavior:

screen shot 2018-10-02 at 4 26 07 pm

Added a comment in the code

File: horizontal-picker/src/main/java/com/github/jhonnyx2012/horizontalpicker/HorizontalPickerRecyclerView.java (hunk 2 above)
Comment (GuillaumeDelente) [reply horizontal-picker/src/main/java/com/github/jhonnyx2012/horizontalpicker/HorizontalPickerRecyclerView.java]: Probably by playing with the fling manager or something, haven't had the time to look at it.

**Pull request merged**